frontend/src/v5/services/realtime/realtime.service.ts

```diff
@@ -0,0 +1,74 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { io, Socket } from 'socket.io-client';
+
+const CHAT_SERVER = 'http://local.3drepo.io:3000';
+
+interface IRoomType {
```

Comment:
I don't know whether you want to think about this now,

but as it stands, we have 2 rooms

{ teamspace, project, model } - for model related notifications
{ notification: true } - for user's notifications

So technically everything is optional here